code/modules/projectiles/projectile/hookshot.dm

```diff
@@ -244,3 +244,99 @@
 		sharpness = 2
 	..(A)
 
+//Wind-up Boxes///////////////////////////////////////////////////////////////////
+
+/obj/item/projectile/hookshot/whip/bootbox
+	name = "boot-in-a-box"
+	icon_state = "spring"
+	icon_name = "spring"
+	nodamage = 0
+	damage = 0
+	sharpness = 0
+	kill_count = 20 //range is defined by maxlength so this prevents animation issues
+	failure_message = ""
+	can_tether = FALSE
+	var/windUp = 0
+	var/springForce = 0
+
+/obj/item/projectile/hookshot/whip/bootbox/OnFired()
```

Comment:
Same comment as above, I see no reason for bootboxes and clownboxes to be so different.